src/waveform/renderers/waveformwidgetrenderer.h

```diff
@@ -74,27 +82,54 @@ class WaveformWidgetRenderer {
         return m_trackPixelCount * (position - m_firstDisplayedPosition);
     }
 
-    double getPlayPos() const { return m_playPos;}
+    double getPlayPos() const {
+        return m_playPos;
+    }
     int getPlayPosVSample() const {
         return m_playPosVSample;
     }
     int getTotalVSample() const {
         return m_totalVSamples;
     }
-    double getZoomFactor() const { return m_zoomFactor;}
-    double getGain() const { return m_gain;}
-    int getTrackSamples() const { return m_trackSamples;}
+    double getZoomFactor() const {
+        return m_zoomFactor;
+    }
+    double getGain() const {
+        return m_gain;
+    }
```

Comment:
Not your fault, but shouldn't this be `CSAMPLE_GAIN` instead of `double`? Anyway, fixing this is out of scope for this PR.